### ISSUE TITLE
Model Training & Hyperparameter Tuning

### ISSUE THREAD

**Issue by jinujung1021** [OREL-group/Project-Management#111]
Currently training Random Forest and XGBoost models & Logistic Regression

1. Running GridSearchCV
2. Comparing accuracy/recall
3. Checking performance differences

This is the main task being worked on right now.
